subproviders/hooked-wallet.js

```diff
@@ -60,33 +69,42 @@ HookedWalletSubprovider.prototype.handleRequest = function(payload, next, end){
       return
 
     case 'eth_sendTransaction':
-      var txParams = payload.params[0]
-      async.waterfall([
-        self.validateTransaction.bind(self, txParams),
-        self.approveTransaction.bind(self, txParams),
-        self.checkApproval.bind(self),
-        self.finalizeAndSubmitTx.bind(self, txParams)
-      ], end)
+      self.managesAccount(payload.params[0].from, function(err, manages){
+        if (err) return cb(err)
+        if(manages){
+          var txParams = payload.params[0]
+          async.waterfall([
+            self.validateTransaction.bind(self, txParams),
+            self.approveTransaction.bind(self, txParams),
+            self.checkApproval.bind(self),
+            self.finalizeAndSubmitTx.bind(self, txParams)
+          ], end)
+        }else{ next() }
```

Comment:
if the account is not managed, would it make more sense to end() with an error describing the reason?

That error could even be sent from the mamagesAccount method, maybe it should be onlyIfManages or something in that case, then you don't need the conditional,  just the error handling.